jqassistant/index.adoc

```diff
@@ -26,6 +26,39 @@ LIMIT 1
 ----
 // end::designsmellGod[]
 
+==== Passive Abstraction (Data class)
+// tag::designsmellPassiveAbstraction[]
+[[designsmell:PassiveAbstraction]]
+[source,cypher,role=concept]
+.A class declares fields without any methods operating on it.
+----
+MATCH
+  (field:Field)<-[:DECLARES]-(class:Class)-[:DECLARES]->(method:Method)
+WHERE NOT
+  (method)-[:READS|:WRITES]->(field)
+AND
+  (class)<-[:CONTAINS]–(:Package {name: "passive"})
```

Comment:
OK, please do it this way: In the "simple" version, it's no problem when also other smells occur in the result list. So you can now remove this line.

Reply:
> OK, please do it this way: In the "simple" version, it's no problem when also other smells occur in the result list. So you can now remove this line.

removed.